src/jit/standalone/CMakeLists.txt

```diff
@@ -4,6 +4,11 @@ add_definitions(-DSELF_NO_HOST)
 add_definitions(-DFEATURE_READYTORUN_COMPILER)
 remove_definitions(-DFEATURE_MERGE_JIT_AND_ENGINE)
 
+if(CLR_CMAKE_PLATFORM_LINUX)
```

Comment:
I saw the similar thing in other places like mscoree where the following check is used.
Should we do similar thing here or would it be enough? 
```
   if(CMAKE_SYSTEM_NAME STREQUAL Linux OR CMAKE_SYSTEM_NAME STREQUAL FreeBSD OR CMAKE_SYSTEM_NAME STREQUAL NetBSD)
```